toolkit/scripts/toolchain/container/toolchain-remote-wget-list

```diff
@@ -39,7 +39,6 @@ http://ftp.gnu.org/gnu/readline/readline-7.0.tar.gz
 http://ftp.gnu.org/gnu/tar/tar-1.30.tar.xz
 http://ftp.gnu.org/gnu/texinfo/texinfo-6.5.tar.xz
 https://www.cpan.org/src/5.0/perl-5.30.3.tar.gz
-https://github.com/microsoft/WSL2-Linux-Kernel/archive/linux-msft-5.4.91.tar.gz
```

Comment:
Shouldn't this be now replaced with the new link?

Reply:
Unfortunately I cannot just update the link here.

The eventual download link is https://github.com/microsoft/CBL-Mariner-Linux-Kernel/archive/rolling-lts/mariner/5.10.13.1.tar.gz which, when downloaded with our standard wget command, will yield `5.10.13.1.tar.gz`. Because this tarball is only a version number, there are possible naming collisions on the source server. So to deal with this, I added a new wget command to the dockerfile to wget the source and rename the output with the `-O` option